test/test_simplecov_json.rb

```diff
@@ -51,7 +51,8 @@ def test_format
     # lines_of_code
     assert_equal(formatter.format(result), {
       'timestamp' => created_at.to_i, 
-      'command_name' => 'RSpec', 
+      'command_name' => 'RSpec',
+      'project_root' => SimpleCov.root,
```

Comment:
I tried stubbing this in the test but it failed because [SimpleCov tries to `mkdir` "#{SimpleCov.root}/coverage".](https://github.com/colszowka/simplecov/blob/master/lib/simplecov/configuration.rb#L45) Overriding `Simplecov.root` with a random directory name will cause that dir to be created there, or error out if the test user doesn't have write permission.